generated/rflx-rflx_generic_types.ads

```diff
@@ -42,17 +44,16 @@ package RFLX.RFLX_Generic_Types is
    type Offset is mod 8;
 
    generic
-      type Value is (<>);
+      type Value is mod <>;
    function Extract (Data : Bytes;
                      Ofst : Offset) return Value with
```

Comment:
Can we use `Offset` as the parameter name here?. In the parameter list this should be possible by using the fully qualified name `Offset : RFLX.RFLX_Generic_Types`), but I'm unsure about the contracts...

Reply:
It's possible, but I think it decreases the readability. Maybe it's just me, but I find it harder to distinguish long names.

```Diff
    generic
       type Value is mod <>;
-   function Extract (Data : Bytes;
-                     Ofst : Offset) return Value with
+   function Extract (Data   : Bytes;
+                     Offset : RFLX_Generic_Types.Offset) return Value with
      Pre =>
-       ((Offset'Pos (Ofst) + Value'Size - 1) / Byte'Size < Data'Length
-        and then (Offset'Pos (Ofst) + Value'Size - 1) / Byte'Size <= Natural'Size
-        and then (Byte'Size - Natural (Offset'Pos (Ofst) mod Byte'Size)) < Long_Integer'Size - 1);
+       ((RFLX_Generic_Types.Offset'Pos (Offset) + Value'Size - 1) / Byte'Size < Data'Length
+        and then (RFLX_Generic_Types.Offset'Pos (Offset) + Value'Size - 1) / Byte'Size <= Natural'Size
+        and then (Byte'Size - Natural (RFLX_Generic_Types.Offset'Pos (Offset) mod Byte'Size)) < Long_Integer'Size - 1);
 
    generic
       type Value is mod <>;
-   procedure Insert (Val  :        Value;
-                     Data : in out Bytes;
-                     Ofst :        Offset) with
+   procedure Insert (Val    :        Value;
+                     Data   : in out Bytes;
+                     Offset :        RFLX_Generic_Types.Offset) with
      Pre =>
-       (Offset'Pos (Ofst) + Value'Size - 1) / Byte'Size < Data'Length;
+       (RFLX_Generic_Types.Offset'Pos (Offset) + Value'Size - 1) / Byte'Size < Data'Length;
```

Do you still prefer it that way?

Reply:
No, that hasn't really improved overall readability. We still use a different name for the offset parameter nonetheless - I just don't associate `Ofst` with *Offset*. Suggestion: `Off` or `Offst`.

Reply:
Fixed by f954782.